src/Ledger/Transaction.lagda

```diff
@@ -66,7 +66,7 @@ the transaction body are:
   field crypto : _
   open Crypto crypto public
   open Ledger.TokenAlgebra ScriptHash public
-  open Ledger.Address Network KeyHash ScriptHash public
+  open Ledger.Address Network KeyHash ScriptHash ⦃ it ⦄ ⦃ it ⦄ ⦃ it ⦄ public
```

Comment:
Is there a reason you added these? It works for me without them. Does that break with the new instance resolution?

Reply:
It broke some stuff for Haskell type generation.

Module applications don't add implicit/instance argument at the end, so without this, the types you get all take instance arguments that are resolved at use site, so when you want to generate a Haskell type for `Credential` say, the tactic needs to deal with types parameterised by instance arguments.

Reply:
Ah, makes sense. Thank you!